docs/fundamentals/write-operations.txt

```diff
@@ -0,0 +1,608 @@
+
+      {
+        "_id": "660e...",
+        "performer": "Mitsuko Uchida",
+        "genres": [],
+        "updated_at": ...,
+        "created_at": ...
+      }
+
+
+.. _laravel-modify-documents-update-array-values:
+
+Update the Value of an Array Element Example
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This section shows how to use the ``$`` positional operator to update specific
+array elements in a MongoDB document. The ``$`` operator represents the first
+array element that matches the query. The following code example shows the
+structure of a positional operator update call on a single matching document:
+
+
+.. note::
+
+   Currently, {+odm-short+} offers this operation only on the ``DB`` facade
+   and not on the Eloquent ORM.
+
+.. code-block:: none
+   :copyable: false
+
+   DB::connection('mongodb')
+      ->getCollection(<collection name>)
+      ->updateOne(
+          <match criteria>,
+          ['$set' => ['<array field>.$' => <replacement value>]]);
+
+
+The following example shows how to replace the array value ``"dance-pop"``
+with ``"contemporary"`` in the ``genres`` array field. Click the
+:guilabel:`VIEW OUTPUT` button to see the updated document:
+
+.. io-code-block::
+
+   .. input:: /includes/fundamentals/write-operations/WriteOperationsTest.php
+      :language: php
+      :dedent:
+      :start-after: begin model array positional
+      :end-before: end model array positional
+
+   .. output::
+      :language: json
+      :visible: false
+
+      {
+        "_id": "660e...",
+        "performer": "Mitsuko Uchida",
+        "genres": [
+          "classical",
+          "contemporary"
+        ],
+        "updated_at": ...,
+        "created_at": ...
+      }
+
+To learn more about array update operators, see :manual:`Array Update Operators </reference/operator/update-array/>`
+in the {+server-docs-name+}.
+
+.. _laravel-fundamentals-delete-documents:
+
+Delete Documents
+----------------
+
+In this section, you can learn how to delete documents from a MongoDB collection
+by using {+odm-short+}. Use delete operations to remove data from your MongoDB
+database permanently.
+
+This section provides examples of the following delete operations:
+
+- :ref:`Delete one document <laravel-fundamentals-delete-one>`
+- :ref:`Delete multiple documents <laravel-fundamentals-delete-many>`
+
+To learn about the Laravel features available in {+odm-short+} that modify
+delete behavior, see the following sections:
+
+- :ref:`Soft delete <laravel-model-soft-delete>`, which lets you mark
+  documents as deleted instead of removing them from the database
+- :ref:`Pruning <laravel-model-pruning>`, which lets you define conditions
+  in which a document should be deleted automatically
+
+.. _laravel-fundamentals-delete-one:
+
+Delete a Document Examples
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can delete one document in the following ways:
+
+- Call the  ``destroy()`` method on the model to delete a document by the
+  primary key.
```

Comment:
```suggestion
- Call the  ``destroy()`` method on the model to delete a document by matching the
  primary key.
```

Reply:
I think the existing sentence and suggestion might be omitting a detail necessary to understand what to pass it, so I'll figure out how to reword.